### PR TITLE
Introduce list todos command

### PR DESCRIPTION
**Pull request by djhi**
- [x] Refactor commands components
- [x] Better tests with helpers
- [x] Add new list todos command

### CODE REVIEW

File: packages/cli/package.json [hunk 1]
```diff
@@ -28,6 +28,7 @@
     "ink-text-input": "^3.1.0",
     "meow": "^5.0.0",
     "react": "^16.8.6",
+    "rimraf": "^2.6.3",
```
Comment (Kmaschta) [inline packages/cli/package.json]: I suggest to put this in the dev dependencies :)

File: packages/cli/src/commands/todo/index.js [hunk 2]
```diff
@@ -2,12 +2,17 @@ import React from 'react';
...
+import ListTodos from './list';
 
-const Todo = ({ command, args, onExit }) => {
+const Todo = ({ command = 'list', args, onExit }) => {
     if (command === 'add') {
         return <AddTodo args={args} onExit={onExit} />;
     }
 
+    if (command === 'list') {
+        return <ListTodos args={args} onExit={onExit} />;
```
Comment (jdemangeon) [inline packages/cli/src/commands/todo/index.js:16]: Can you use consts for this commands too?

**Review by zyhou — APPROVED** [review]

File: packages/cli/src/commands/todo/index.js (hunk 2 above)
Comment (djhi) [reply packages/cli/src/commands/todo/index.js:16]: Actually I was planning to implement an ink version of react-router to handle the commands parsing/selection.

I discovered there was a similar project (https://github.com/jimmed/ink-router) but it has not been updated for a year and is not compatible with the latest ink.

The plan is now to reimplement it at my next hackday :smile:

Comment (jdemangeon) [reply packages/cli/src/commands/todo/index.js:16]: ;)

Comment (djhi) [reply packages/cli/src/commands/todo/index.js:16]: Done

**Pull request merged**